src/logion/controllers/collection.controller.ts

```diff
@@ -184,6 +184,21 @@ export class CollectionController extends ApiController {
             throw badRequest("Cannot replace existing item, you may try to cancel it first");
         }
 
+        const collection = requireDefined(await this.locRequestRepository.findById(collectionLocId));
+        if (collection.collectionCanUpload !== undefined
+            && !collection.collectionCanUpload
+            && body.files !== undefined
+            && body.files.length > 0
+        ) {
+            throw badRequest("Collection cannot upload");
```

Comment:
Maybe have a more explicit message (e.g. `Collection does not accept item files`).

Reply:
Done.